### ISSUE TITLE
Infographic of the main quantum open source projects.

### ISSUE THREAD

**Issue by victor-onofre** [qosf/qosf.org#93]
- The style needs to be aesthetically coherent with the QOSF style 
- The infographic should offer a general overview of the connections between different projects 
- Examples to follow: https://miro.medium.com/v2/resize:fit:1400/0*baf_8nZ3yTHpSCE4

**Comment by Micheallscarn** [comment]
hi , i would love to work on this one , although , where should I get the list of projects from ?

**Comment by ljcamargo** [comment]
Hi @victor-onofre sent a proposal for PR https://github.com/qosf/qosf.org/pull/103 for Unitary Hack 2024

**Comment by victor-onofre** [comment]
Hi @ljcamargo! Thanks for your interest in helping us! It is a good starting point, but many projects are missing. [Here](https://qosf.org/project_list/) you can see a list of different projects, even though the list is outdated. I recommend visiting the [unitary fund discord](https://discord.com/invite/JqVGmpkP96) to check the different projects. 

Hi @Micheallscarn! Part of this issue's challenge is getting a good list of the projects. I share some links that can be helpful, but right now, there are too many quantum open source projects. The idea is to create an infographic to help navigate the different projects. Maybe you can work with @ljcamargo as a team to move faster. 

I don't expect to have the full infographic finished, but it would be nice to have the first version. Once you submit something, please upload the original files to be able to modify the images. 

Let me know if you have any questions. 

Thanks!

**Comment by ljcamargo** [comment]
yes @victor-onofre I used that list, but as I though they would be too much I picked the ones I knew are most common, but I will add many of the others, as I don't know how to rank them by popularity I will add them sequentially. Actually I did the design in illustrator for each project (like cards) with variables from a csv and joined them with a script because I imagined it may need changes or append others, so when can add as many as long as they can fit in a grid. I can share the .csv .ai file and script on the next PR to make quick changes

**Comment by ljcamargo** [comment]
@victor-onofre I sent a recond rendition of this infographic, let me know what do you thinks, https://github.com/qosf/qosf.org/pull/104

**Comment by jyotiraj-code** [comment]
Hi @victor-onofre tried to create a infographic in the latest PR #106.
In the PR: 
- Added .png file to the directory designs
- Followed the dimensions of the "Quantum Computing Map" infographic
- Followed the color theme of QOSF
- Tried to include all the projects but still need revisions
- Followed the list from QOSF.org website
- Excluded deprecated, abandoned, not maintained now and very old projects

**Comment by jyotiraj-code** [comment]
Hi @ljcamargo I have seen your infographic, great work with it and I really liked the idea of using .csv to udpate the infographic. I am not quite sure how to use this method in Figma (the one I have created in). If we can incorporate that too it will be great.

**Comment by victor-onofre** [comment]
Hi @jyotiraj-code! I left a couple of comments in the PR. Thanks for the help!

**Comment by ljcamargo** [comment]
Hi @victor-onofre , I pushed a new commit with the ai files and scripts I used to generate the infrographic as I previosuly promised. As it is not trivial, I included a file README.md files with brief instructions. You may check the csv with the included projects and if some changes are made I may fetch then and provide the rendering again. Also included the project images in png, (including some placholder github project designs for the projects that doesn't have logo) if they are in any help for you. Please let me know if anything else will be needed to close this PR for Unitary Hack 2024 which closes in a couple of days
